package.json

```diff
@@ -26,17 +26,21 @@
   },
   "engines": {
     "node": ">=16.0.0"
-},
+  },
   "dependencies": {
     "@actions/core": "^1.6.0",
     "@actions/github": "^5.0.0",
     "@octokit/action": "^3.18.0",
+    "@types/filesystem": "^0.0.32",
     "axios": "^0.25.0",
+    "file-system": "^2.2.2",
```

Comment:
I don't know what this package is and you don't seem to be using it so it should be removed